src/cloud/cdn/configure-fastly.md

```diff
@@ -123,41 +122,126 @@ To upload the Fastly VCL:
+Check with your [DNS registrar](https://lookup.icann.org/) for information about updating the DNS configuration.
+
+**Prerequisites:**
+
+-  Enable the Fastly module.
+-  Upload the default Fastly VCL code.
+-  Provide a list of top-level and subdomains for each environment to your Customer Technical Advisor (CTA) or submit them in a Magento Support ticket.
+-  Wait for confirmation that the specified domains have been added to your Cloud environments.
+-  On Starter projects, add the domains to your Fastly service configuration. See [Manage domains]({{ site.baseurl }}/cloud/cdn/configure-fastly-customize-cache.html#manage-domains).
+
+{:.procedure}
+To update your DNS configuration for development:
 
-We recommend a significantly lower TTL value when switching the DNS record. This value tells the DNS how long to cache the DNS record. When shortened, it refreshes the DNS faster. For example, you can change the TTL value from 3 days to 10 minutes when you are testing your site. Be advised that shortening the TTL value  adds load to the web server.
+1. Add CNAME records to point pre-production URLs to the Fastly service: `prod.magentocloud.map.fastly.net`, for example:
 
-After checking with your registrar about where to change your DNS settings, add a CNAME record for your website that points to the Fastly service: `prod.magentocloud.map.fastly.net`. If you use multiple hostnames for your site, you must add a CNAME record for each one.
+   | Domain or Subdomain | CNAME
+   |---------------------|------
+   | mcprod.your-domain.com | prod.magentocloud.map.fastly.net
+   | mcstaging.your-domain.com | prod.magentocloud.map.fastly.net
 
-CNAME records cannot be set for apex domains, also referred to as a naked or base domains. You must use `A` records for this.
-`A` records map a domain name to the following Fastly IP addresses:
+   When the CNAME records are live, Magento provisions certificates and uploads the SSL/TLS certificates.
 
--  `151.101.1.124`
--  `151.101.65.124`
--  `151.101.129.124`
--  `151.101.193.124`
+1. Add ACME challenge CNAME records for domain validation and pre-provisioning of Production SSL/TLS certificates, for example:
 
-Refer to [Go live checklist]({{ site.baseurl }}/cloud/live/site-launch-checklist.html) for more information.
+   | Domain or Subdomain  | CNAME
+   |----------------------|---------
+   | _acme-challenge.your-domain.com<br>| 0123456789abcdef.validation.magento.cloud
+   | _acme-challenge.www.your-domain.com<br> | 9573186429stuvwx.validation.magento.com
+   | _acme-challenge.mystore.your-domain.com<br> | 1234567898zxywvu.validation.magento.cloud
+   | _acme-challenge.subdomain.your-domain.com<br>| 1098765743lmnopq.validation.magento.cloud
 
-### TLS and Fastly
+   When the CNAME records are live, validates the domains and provisions the SSL/TLS certificates. These certificates will be applied to your environment when you update the DNS configuration to point your production domains to Fastly.
```

Comment:
This paragraph has a few problems. Something seems missing before the word "validates", and there is a "will be" phrase that could be improved. Maybe it's something "Magento" does like in the previous step? Magento validates ... and applies ...?

```suggestion
   After adding the CNAME records, Magento validates the domains and provisions the SSL/TLS certificates.= and applies these certificates to your environment when you update the DNS configuration to point your production domains to Fastly.
```